src/styles/base/_root.scss

```diff
@@ -6,6 +6,7 @@
 }
 
 body {
+	height: 100vh;
```

Comment:
moet dit geen min-height zijn? Anders gaan we issues hebben met pagina's die overflow hebben denk ik.

Reply:
For some reason werkt de lockScroll hook alleen zo en kan ik nog perfect scrollen zo, maar ik test en dubbelcheck nog!

Reply:
Tested op Chrome, Firefox en Safari. dat heeft geen impact op scrollen op bv. de homepagina en met `min-height` werkt de hook niet for some reason. Als het later voor problemen zorgt kunnen we deze property ook door de hook laten invullen 🤷